config/jobs/image-pushing/k8s-staging-cluster-addons.yaml

```diff
@@ -0,0 +1,27 @@
+postsubmits:
+  kubernetes-sigs/cluster-addons:
+    - name: cluster-addons-postsubmit-push-to-staging
+      cluster: k8s-infra-prow-build-trusted
+      annotations:
+        testgrid-dashboards: sig-cluster-lifecycle-cluster-addons
+      decorate: true
+      branches:
+        - ^master$
+        - ^release-.*
+        # Build tags also
+        # this is a regex for semver, from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+        - ^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$
```

Comment:
that's a wicked regex!

![image](https://user-images.githubusercontent.com/23304/91461542-2348e500-e857-11ea-9644-e722a2dfc274.png)

Reply:
It sure is ... I copied and pasted it from someone else, who copied and pasted it from someone else, etc until we get to someone who was probably just trolling :-)

Reply:
LOL +1